cmd/migrate/kafka-acls.go

```
@@ -0,0 +1,76 @@
+		return nil
+	}
+
+	saramaCfg, err := kafka.GetSaramaConfig(&cfg.Kafka)
+	if err != nil {
+		return fmt.Errorf("build kafka client config: %w", err)
+	}
+	admin, err := sarama.NewClusterAdmin(cfg.Kafka.Brokers, saramaCfg)
+	if err != nil {
+		return fmt.Errorf("connect cluster admin: %w", err)
+	}
+	defer admin.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `admin.Close` is not checked

(errcheck)